resources/views/accessories/view.blade.php

```diff
@@ -88,17 +82,154 @@ class="table table-striped snipe-table"
+            </li>
+            @endif
+
+            @if ($accessory->category)
+            <li>
+                <strong>{{ trans('general.category') }}: </strong>
+                @can('view', \App\Models\Category::class)
+                  <a href="{{ route('categories.show', $accessory->category->id) }}">{{ $accessory->category->name }}</a>
+                @else
+                  {{ $accessory->category->name }}
+                @endcan
+            </li>
+            @endif
+
+            @if ($accessory->location)
+            <li><strong>{{ trans('general.location') }}: </strong>
+                @can('superuser')
+                    <a href="{{ route('locations.show', ['location' => $accessory->location->id]) }}">
+                    {{ $accessory->location->name }}
+                    </a>
+                @else
+                    {{ $accessory->location->name }}
+                @endcan
+            </li>
+            @endif
+
+            <br />
+            <li>
+                <strong>{{ trans('admin/accessories/general.total') }}: </strong>
+                {{ $accessory->qty }}
+            </li>
+            <li>
+                <strong>{{ trans('general.checkouts_count') }}: </strong>
+                {{ $accessory->qty - $accessory->numRemaining() }}
+            </li>
+            <li>
+                <strong>{{ trans('admin/accessories/general.remaining') }}: </strong>
+                {{ $accessory->numRemaining() }}
+            </li>
+            <li>
+                <strong>{{ trans('general.min_amt') }}: </strong>
+                {{ $accessory->min_amt }}
+            </li>
+
+
+            @if ($accessory->supplier)
+            <br />
+            <li>
+                <strong>{{ trans('general.supplier') }}</strong>
+                <br />
+                {{ $accessory->supplier->name }}
+                <br />
+                
+                @if (($accessory->supplier->url))
+                    <i class="fa fa-globe"></i> <a href="{{ $accessory->supplier->url }}">{{ $accessory->supplier->url }}</a>
+                    <br />
+                @endif
+
+                @if (($accessory->supplier->contact))
+                    <i class="fa fa-user"></i> <a href="{{ $accessory->supplier->contact }}">{{ $accessory->supplier->contact }}</a>
+                    <br />
+                @endif
+
+                @if (($accessory->supplier->phone))
+                    <i class="fa fa-phone"></i><a href="tel:{{ $accessory->supplier->phone }}">{{ $accessory->supplier->phone }}</a>
+                    <br />
+                @endif
+
+                @if (($accessory->supplier->email))
+                    <i class="fa fa-envelope"></i> <a href="mailto:{{ $accessory->supplier->email }}">{{ $accessory->supplier->email }}</a>
+                    <br />
+                @endif
+            </li>
+            @endif
+
+            @if ($accessory->manufacturer)
```

Comment:
Same here for manufacturer? Hrm. Might be okay - the accessory view screen is awful spartan.